config/routes.rb

```diff
@@ -7,4 +7,16 @@
     end
   end
   root to: "users#index"
+
+  namespace :api do
+    namespace :v1 do
+      post :auth, to: "authentication#login"
+      resources :users do
+        post :create, to: "users#create"
+        resources :posts do
+          resources :comments
```

Comment:
- [ ] [OPTIONAL] It would be helpful to add the API ENDPOINTS to your README file so your API is well documented